src/Common/HashTable/Hash.h

```diff
@@ -54,6 +56,8 @@ inline DB::UInt64 intHashCRC32(DB::UInt64 x)
     return _mm_crc32_u64(-1ULL, x);
 #elif defined(__aarch64__) && defined(__ARM_FEATURE_CRC32)
     return __crc32cd(-1U, x);
+#elif defined(ARCH_PPC64LE)
```

Comment:
A search for `__crc32cd` gives 11 locations in the code - all of them must be extended for PowerPC support.

Reply:
This fix concentrate only about the WeakHash32 Collision test failure.  We will have a check whether the changes needed for all the other places as well.

Reply:
instead of all this strange ARCH_PPC64LE stuff.. why not use compiler macros?
```
#elif (defined(__PPC64__) || defined(__powerpc64__)) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Reply:
@MeenaRenganathan22 we should handle all occurrences of __crc32cd in the code base, we will have to deal with it anyway

Reply:
Will make changes to handle all occurrences of __crc32cd